config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -159,6 +148,8 @@ presubmits:
     always_run: false
     optional: true
     decorate: true
+    decoration_config:
+      timeout: 4h
```

Comment:
Update the timeout to avoid issues that were seen with the periodics

Reply:
should the interval also be set with this increase in timeout?

Reply:
Interval doesn't need to be set because it is a presubmit job. Only need intervals on the periodics.